tests/util/test_xsDateTime.py

```diff
@@ -0,0 +1,41 @@
+
+from datetime import datetime, timedelta
+from codalib.util import xsDateTime_parse, xsDateTime_format, XSDateTimezone
+
+
+def test_parse_date():
+
+    dt_str = "2017-01-27T14:43:00+00:00"
+
+    assert isinstance(xsDateTime_parse(dt_str), datetime)
+    dt = xsDateTime_parse(dt_str)
+    assert dt.utcoffset() == timedelta(0)
+
+def test_parse_date2():
```

Comment:
I personally think a name such as `test_parse_date_with_nonzero_offset` would be more informative. Along those lines, maybe the previous test could be similarly named to describe what's different about the two?